### ISSUE TITLE
Loki addon not registered. Add `import 'loki/configure-react' when running loki for ./storybook-static

### ISSUE THREAD

**Issue by alexandrzavalii** [oblador/loki#202]
I am not able to run loki for static storybook, it works for localhost though.

I am using this command from examples
```
    "loki": "./node_modules/.bin/loki --requireReference --reactUri file:./storybook-static",
```
and I get this error
```
Loki addon not registered. Add `import 'loki/configure-react'` to your config.js file.
```
these are my settings for loki in package.json
```
  "loki": {
    "diffingEngine": "gm",
    "chromeTolerance": 1,
    "reactUri": "file:./storybook-static",
    "configurations": {
      "chrome.laptop": {
        "target": "chrome.app",
        "width": 1366,
        "height": 768,
        "deviceScaleFactor": 1,
        "mobile": false
      }
    }
  }
```

**Comment by oblador** [comment]
Do you have a `.storybook/config.js`? If you're running a newer version of storybook, you might need to create a `.storybook/preview.js` and add `import 'loki/configure-react'` in there

**Comment by oblador** [comment]
Fixed in #225
